src/matrix.js

```diff
@@ -23,7 +23,6 @@
 var UI = {
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```

Comment:
Is this change necessary in this PR?

Reply:
Not strictly but it is related and life is too short to separate them